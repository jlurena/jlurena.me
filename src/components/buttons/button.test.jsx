/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '.'

describe('<Button/>', () => {
  const className = 'someclass'
  const component = renderer.create(
    <Button className={className}><p>Hello</p></Button>
  )

  const tree = component.toJSON()

  test('Rendering', () => {
    expect(tree).toMatchSnapshot()
  })
})
