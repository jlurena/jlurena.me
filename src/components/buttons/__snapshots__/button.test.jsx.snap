// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> Rendering 1`] = `
<button
  className="someclass btn"
  type="button"
>
  <p>
    Hello
  </p>
</button>
`;
